docker/build_container.sh

```diff
@@ -28,9 +28,9 @@ DOCKER_TARGET=${DOCKER_TARGET:-"runtime"}
 DOCKER_EXTRA_ARGS=${DOCKER_EXTRA_ARGS:-""}
 
 # Build args
-CUDA_MAJOR_VER=${CUDA_MAJOR_VER:-11}
-CUDA_MINOR_VER=${CUDA_MINOR_VER:-8}
-CUDA_REV_VER=${CUDA_REV_VER:-0}
+CUDA_MAJOR_VER=${CUDA_MAJOR_VER:-12}
```

Comment:
I think we can also delete the references to `RAPIDS_VERSION` and `TENSORRT_VERSION` in this file.